src/Swashbuckle.AspNetCore.SwaggerUI/index.html

```diff
@@ -102,6 +102,11 @@
             if (interceptors.ResponseInterceptorFunction)
                 configObject.responseInterceptor = parseFunction(interceptors.ResponseInterceptorFunction);
 
+            // Allow adjusting config through injected JavaScript files
+            if(typeof initConfig === "function") {
```

Comment:
Is there a better way of hooking this up as a global that's more robust and/or more likely to not pollute with any existing objects (e.g. what if it was called `swashbuckleInitConfig` for instance)?